docs/release-process.md

```diff
@@ -2,96 +2,78 @@
 
 This guide illustrates how to perform a release for kop.
 
-## Making the release
+## Naming convention
 
-The steps for releasing are as follows:
+KoP should following the naming convention:
 
-1. Prepare for a release
-2. Create the release branch
-3. Update project version and tag
-4. Write a release note
-5. Move master branch to next version
-6. Announce the release
-
-## Steps in detail
-
-1. Prepare for a release
-
-Create a new milestone and move the pull requests that can not
-be published in this release to the new milestone.
+- Branch name: `branch-X.Y.Z(.M)`
+- Tag name: `vX.Y.Z(.M)`(stable)
+- Release candidate tag: `vX.Y.Z(.M)-rc-$(date +%Y%m%d%H%M)` (unstable)
 
-2. Create the release branch
+`(.M)` means our internal version release number, most of our repository is an extensions/tools for apache/pulsar. To keep track of the repository is produced by which version of the apache/pulsar, we will carry the apache/pulsar version number and using the `.M` to represent our internal release version. And all repository should keep align with streamnative/pulsar.
 
-We are going to create a branch from `master` to `branch-x.y`
-where the tag will be generated and where new fixes will be
-applied as part of the maintenance for the release. `x.y.z`
-is the version of the release.
+There has two type of the tags, one is stable `vX.Y.Z(.M)`, and another is unstable `vX.Y.Z(.M)-rc-$(date +%y%m%d%H%M)`. A stable tag represent this release is a verified release, and an unstable tag represent this release is not verified. 
 
-The branch needs only to be created when creating major releases,
-and not for patch releases.
+## Release workflow
```

Comment:
Sorry, maybe I haven't clearly pointed out, all I mentioned in this article https://streamnative.slab.com/posts/repository-release-template-prb0lvyt is a **template**. I just want to make a template and we can apply it to any of our repositories. According to the release template, we should have a clear release process.

I see you just copy it here, does this workflow can release kop step by step?

Reply:
Ok, I'll push the fix soon